plugin-flex-ts-template-v2/src/types/task-router/Worker.ts

```diff
@@ -21,6 +21,7 @@ export default interface Worker extends EventEmitter {
   workspaceSid: string;
 }
 
+// flex-ui also defines this interface, but doesn't include our custom attributes.
```

Comment:
Im wondering if this interface should then extend the existing interface and just add our custom attributes.  So we use CustomWorkerAttributes or something?